includes/templates/admin/edit-story.php

```diff
@@ -76,6 +76,12 @@
 
 ?>
 
+<div id="stories-editor-no-js" class="wrap">
+	<div class="notice notice-error notice-alt">
+		<p><?php esc_html_e( 'The Web Stories editor requires JavaScript. Please enable JavaScript in your browser settings.', 'web-stories' ); ?></p>
```

Comment:
This way there's only 1 string to translate:

```suggestion
		<p><?php esc_html_e( 'Web Stories for WordPress requires JavaScript. Please enable JavaScript in your browser settings.', 'web-stories' ); ?></p>
```